src/main/java/org/retest/rebazer/connector/GithubConnector.java

```diff
@@ -61,7 +61,13 @@ public PullRequest getLatestUpdate( final PullRequest pullRequest ) {
 	@Override
 	public boolean isApproved( final PullRequest pullRequest ) {
 		final DocumentContext jsonPath = jsonPathForPath( requestPath( pullRequest ) + "/reviews" );
-		return jsonPath.<List<String>> read( "$..state" ).stream().anyMatch( "APPROVED"::equals );
+		final List<String> reviews = jsonPath.<List<String>> read( "$..state" );
+		final String titleAndDescription = pullRequest.getTitle().concat( pullRequest.getDescription() );
```

Comment:
create method `pullRequest.reviewByAllReviewersRequested()` to reduce code duplication.